packages/core/frontend/sirius-components-omnibox/src/Omnibox.types.ts

```diff
@@ -11,14 +11,14 @@
  *     Obeo - initial API and implementation
  *******************************************************************************/
 
-import { GQLOmniboxCommand } from './useWorkbenchOmniboxCommands.types';
+import { RefObject } from 'react';
 
 export interface OmniboxProps {
   open: boolean;
   loading: boolean;
-  commands: GQLOmniboxCommand[] | null;
+  commands: OmniboxCommand[] | null;
   onQuery: (query: string, mode: OmniboxMode) => void;
-  onCommandClick: (command: GQLOmniboxCommand, mode: OmniboxMode) => void;
+  onCommandClick: (command: OmniboxCommand, mode: OmniboxMode, input: RefObject<HTMLInputElement>) => void;
```

Comment:
It's very odd to provide access to the input like that, in terms of responsibility it feels like the inverse of what we would do. Ref are used for a child component to communicate with its parent by giving control to the parent. Here, you are giving back to the parent something that it should control already.